client/src/App.js

```diff
@@ -2,11 +2,13 @@ import React from "react";
 
 import Header from "./components/Header";
 import "./App.css";
+import ToDoList from "./components/ToDoList";
 
 function App() {
     return (
         <div className="App">
             <Header />
+            <ToDoList />
```

Comment:
This can be done later and should be included in @Chuppay's Body component

Reply:
^yeah let's try avoid conflicts and not alter the App.js file.